audit-linux.c

```diff
@@ -61,7 +61,7 @@ linux_audit_record_event(int uid, const char *username, const char *hostname,
 	 * Do not report error if the error is EPERM and sshd is run as non
 	 * root user.
 	 */
-	if ((rc == -EPERM) && (geteuid() != 0))
+	if ((rc == -EPERM) && (geteuid() != ROOT_UID))
```

Comment:
this one is in the linux specific auditing code, I don't think it's applicable to you?

Reply:
It is, the effective UID and initiating UID is definitely a factor on the platform. In older ports, we have had people complaining (and getting caught) for violating this. It is a real and frequent thing on NonStop.

Reply:
yeah but this specific instance is in Linux specific code and the whole file is inside #if defined(USE_LINUX_AUDIT.  Are you setting that?

Reply:
Will revert this one, although would it not make sense to be consistent about the UID across the entire code?